### PR TITLE
Instagram icon on footer

### PR DESCRIPTION
**Pull request by tytotf08**
Added svg to Instagram link in footer. Now the link is a black circle with the ig icon in it.

### CODE REVIEW

**Comment by tytotf08** [comment]
Does #5

File: css/screen.scss [hunk 1]
```diff
@@ -140,6 +140,16 @@ body.info {
   color: #888;
   font-size: 14px;
   font-weight: 100;
+  a {
+    display: flex;
```
Comment (shawnliu) [inline css/screen.scss]: Do you know why you have this declaration?

Also I think this makes the a href element span the entire width of the page (if you don't understand what I mean, ask me in person).

**Comment by tytotf08** [comment]
Fixed some issues with the PR!

File: css/screen.scss [hunk 2]
```diff
@@ -140,6 +140,13 @@ body.info {
   color: #888;
   font-size: 14px;
   font-weight: 100;
+  a {
+    width: 30px;
```
Comment (shawnliu) [inline css/screen.scss:144]: Do you need this?

Comment (tytotf08) [reply css/screen.scss:144]: That is how I don't make the link span the whole page - make it the same width as the svg.

Comment (shawnliu) [reply css/screen.scss:144]: Are you sure you need it?

Comment (tytotf08) [reply css/screen.scss:144]: Yes

**Review by shawnliu — APPROVED** [review]
Good. Ok. Fine. Acceptable.

**Pull request merged**